src/App/Fossa/Container.hs

```diff
@@ -260,7 +261,7 @@ syftCommand :: BinaryPaths -> ImageText -> Command
 syftCommand bin (ImageText image) =
   Command
     { cmdName = pack . toFilePath $ toExecutablePath bin,
-      cmdArgs = ["-o", "json", image],
+      cmdArgs = ["--scope", "all-layers", "-o", "json", image],
```

Comment:
Can we post a short comment on these options, and also maybe link to syft docs explaining more if needed?

Reply:
Add comments - the docs are not particularly useful so I left them out. if the docs update, I'll add them.